src/App.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import NekaKomponenta from './components/NekaKomponenta';
-import './App.css';
+import NavbarPage from './components/NavbarPage';
+import ModalLogin from './components/ModalLogin';
+import ModalRegister from './components/ModalRegister.js';
 
+import './App.css';
 function App() {
   return (
     <div className="App">
       <NekaKomponenta/>
-      
+      <NavbarPage/>
+      <ModalLogin/>
+      <ModalRegister/>
     </div>
```

Comment:
nemoj ovako u app komponenti treba da bude router u startu podesen
            <Router history={history}>
                <Header />
                <Route path="/login" component={LogInDialog} />
                <Route path="/signUp" component={SignUpDialog} />
            </Router>
kao ovde
to je da bi mogli da redirektujemo korisnika ako oce negde da ode gde ne sme

Reply:
ne vidi se kod koji sam ti poslao